### PR TITLE
"What is a thumbnail?"

### PR DESCRIPTION
**Pull request by indiegamehunt**

### CODE REVIEW

File: what-thumbnail/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## What is a thumbnail?
+
+Your game's thumbnail is probably the most important image of your game on the site. This is the image that will appear on the Newly Added list, in search results, and so on.
+
+It's hard to stress the importance of thumbnails enough. These little images can be the difference between someone checking out your game and barely noticing it. In many cases, a thumbnail will be the first impression someone has of your game.
+
+[Choosing a thumbnail image](Link)
```
Comment (MaskofThoro) [inline what-thumbnail/index.md]: this look a little weird as a statement vs question.

**Comment by MaskofThoro** [comment]
combine this with #25

**Comment by indiegamehunt** [comment]
Combining #24, #25, and #26

**Comment by indiegamehunt** [comment]
Combined with dev18 (#26)

**Pull request closed without merging**